locale/en/site.json

```diff
@@ -163,15 +163,11 @@
   },
   "banners": {
     "index": {
-      "startDate": "2022-01-04T16:00:00.000Z",
-      "endDate": "2022-01-25T16:00:00.000Z",
-      "text": "New security releases now available for Node.js 12, 14, 16, and 17 release lines",
-      "link": "https://nodejs.org/en/blog/vulnerability/jan-2022-security-releases/"
-    },
-    "blacklivesmatter": {
-      "visible": false,
-      "text": "#BlackLivesMatter",
-      "link": "black-lives-matter/"
+      "startDate": "2022-02-28T16:00:00.000Z",
+      "endDate": "2022-03-21T16:00:00.000Z",
+      "rawHTML": "<p class=\"home-version home-version-banner\" style=\"font-size: large; text-align: left; line-height: 1.25; \">We stand with the people of Ukraine. We encourage compassion, and hope for peace. Please support humanitarian efforts for the Ukraine crisis through the <a href=\"https://www.icrc.org/en/donate/ukraine\">International Committee of the Red Cross</a>. #StandWithUkraine</p>",
```

Comment:
My understanding of the style guide, not sure if preferred by native speakers:

```suggestion
      "rawHTML": "<p class=\"home-version home-version-banner\" style=\"font-size: large; text-align: left; line-height: 1.25; \">We stand with the people of Ukraine. We encourage compassion and hope for peace. Please support humanitarian efforts for the Ukraine crisis through the <a href=\"https://www.icrc.org/en/donate/ukraine\">International Committee of the Red Cross</a>. #StandWithUkraine</p>",
```

Reply:
I would leave out the comma too. However, I'm inclined to use the OpenJS text verbatim. Unless we see it on the OpenJS website without the comma, I'm inclined to keep it.

Reply:
That makes a lot of sense! :)

Reply:
I'm totally fine removing the comma and will remove it from our text. We rarely have consensus on commas at OpenJS. :)

Reply:
Tired: Debating semicolons, tabs, spaces...

Wired: Commas! Let's debate commas!